<launch>
  <remap from="/joy" to="/teleop_joy" />

  <node pkg="joy" type="joy_node" name="joy_node" clear_params="true" output="screen">
  </node>
  <node pkg="drc_task_common" type="joy_to_twist.py" name="joy_to_twist" clear_params="true" output="screen">
  </node>
  <node pkg="drc_task_common" type="teleop_test.l" name="teleop_test" clear_params="true" output="screen">
  </node>
 
</launch>
